packages/calcite-components/src/components/dropdown/dropdown.e2e.ts

```diff
@@ -724,98 +724,169 @@ describe("calcite-dropdown", () => {
     await page.waitForChanges();
     expect(await dropdownWrapper.isVisible()).toBe(false);
   });
+  describe("keyboard support", () => {
```

Comment:
I think tests and describe blocks may need rearranging. This top one is for keyboard support and there's nested ones covering pointer interaction.

Reply:
removed the keyboard support describe block since most of the tests are using `click` to open the dropdown.